### PR TITLE
Optional ean13 country exception.

### PR DESCRIPTION
**Pull request by CanSeeThePain**

### CODE REVIEW

**Comment by barnhill** [comment]
Can I get a description of why this is needed?

**Comment by CanSeeThePain** [comment]
> Can I get a description of why this is needed?
Hi, Brad. Our company uses EAN13 for internal tasks, inside our country. Because of it, the country code is not used. We use the whole number as ID. And some numbers cause "invalid country code" exceptions. But managers say these numbers are necessary. So, without these changes, we can't use nuget package of BarcodeLib to replace our own legacy code.

**Comment by rob313663** [comment]
Hi,

I have a customer that uses EAN-13 for their internal use too. They encode
article number and size code (for clothes) in it. A bit crazy.

/rob

File: BarcodeStandard/Symbologies/EAN13.cs [hunk 1]
```diff
@@ -15,16 +15,23 @@ class EAN13 : BarcodeCommon, IBarcode
...
         private string _countryAssigningManufacturerCode = "N/A";
 
-        public EAN13(string input)
+        public EAN13(string input, bool disableCountryException = false)
         {
             Raw_Data = input;
+			DisableCountryException = disableCountryException;
 
-            CheckDigit();
+			CheckDigit();
```
Comment (barnhill) [inline BarcodeStandard/Symbologies/EAN13.cs:23]: This spacing seems off

**Review by barnhill — APPROVED** [review]

File: BarcodeStandard/Symbologies/EAN13.cs (hunk 1 above)
Comment (barnhill) [reply BarcodeStandard/Symbologies/EAN13.cs:23]: Ill fix this after merging

**Comment by barnhill** [comment]
@rob313663 and @CanSeeThePain,

I just released 2.4.0 to NuGet.  It might take a little bit to show up but its on its way.  The feature you added I modified slightly to allow for disabling of country code parsing and not just throwing the exception away.  This will allow for slightly better efficient generation of EAN-13 codes.  Same general concept but just took it a step further.  I have also allowed for retrieval of the country code via a property since that was overlooked originally.  Thanks for the submission and keep the updates coming!  Keep me posted on how you are using it just for my curiousity!

**Comment by CanSeeThePain** [comment]
> I just released 2.4.0 to NuGet. (...)
Thank you very much! My team leader will be happy to get these changes in the NuGet package :)

**Comment by CanSeeThePain** [comment]
After loading 2.4.0 and adding it to my project, I tried to reproduce the country exception, but to no avail. It seems that now, because all country codes between 0 and 100 filled, at least 2-digits country code will always present. So I think that exception now is unreachable.

**Pull request merged**